src/saveload/saveload.h

```diff
@@ -338,6 +338,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_TABLE_CHUNKS,                       ///< 295  PR#9322 Introduction of CH_TABLE and CH_SPARSE_TABLE.
 	SLV_SCRIPT_INT64,                       ///< 296  PR#9415 SQInteger is 64bit but was saved as 32bit.
 	SLV_LINKGRAPH_TRAVEL_TIME,              ///< 297  PR#9457 v12  Store travel time in the linkgraph.
+	SLV_DOCK_DOCKINGTILES,                  ///< 298  PR#9578 All tiles around docks may be docking tiles.
```

Comment:
maybe already update the line above to "v12-RC1" and this line to v12. While backporting we remove them in both cases, so this will conflict anyway, and saves a change later on :) (assuming this is the last bump in v12 :P)